bindata/daemon/04.daemonset.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         privileged: true
       containers:
       - name: dpu-operator-daemon
-        image: {{.DpuOperatorDaemonImage}}
+        image: localhost:5001/dpu-daemon:dev
```

Comment:
undo - local testing only

Reply:
done - but the ``{{.DpuOperatorDaemonImage}}`` does not work, hence the local change